Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs

```diff
@@ -32,7 +32,15 @@ public void OnToggleHandMesh()
         /// </summary>
         public void OnToggleHandJoint()
         {
-            MixedRealityHandTrackingProfile handTrackingProfile = CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile;
+            MixedRealityHandTrackingProfile handTrackingProfile = null;
+
+            if ((CoreServices.InputSystem != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null))
```

Comment:
Same nit as previous comment, plus the existing third case looks to be a duplicate of the second:

```suggestion
            if (CoreServices.InputSystem?.InputSystemProfile != null)
```